spec/models/receiver_spec.rb

```diff
@@ -6,4 +6,32 @@
 
   it { should validate_presence_of(:letter) }
   it { should validate_presence_of(:institution_id) }
+
+  it "should have a gift" do
+    receiver = create(:receiver)
+    create(:match, receiver: receiver, received: true)
+
+    expect(receiver.number_of_gifts).to eq(1)
+  end
+
+  it "should have a gift" do
```

Comment:
Eh, all these tests have the same description

Reply:
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA